{{cookiecutter.project_shortname}}/misc/header.html

```diff
@@ -0,0 +1,6 @@
+<!--
+  Copyright (C) {{ cookiecutter.year }} {{ cookiecutter.copyright_holder }}.
```

Comment:
Can you take the fixes from [this commit](https://github.com/inveniosoftware/cookiecutter-invenio-module/commit/625fa3c97001499f483e419d9b37e7a0d3599ef8) to have line wrapping.